src/WebUI/Startup.cs

```diff
@@ -53,6 +58,9 @@ public void ConfigureServices(IServiceCollection services)
                 options.UsePkce = true;
             });
 
+            services.AddSingleton<IPostConfigureOptions<OpenIdConnectOptions>, OpenIdConnectPostConfigureOptions>();
```

Comment:
You could also consider using the `AddOptions()` pattern here to avoid having to implement a post-configure action for a single line:

```
services.AddOptions<OpenIdConnectOptions>("oidc")
    .Configure<ISecretsService>((options, secretsService) =>
    {
        // set options with secrets service
    });
```

Reply:
Done that